integrations/docker/images/stage-2/chip-build-cirque/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION=latest
+ARG VERSION=15
```

Comment:
Why every other stage-2 image is based on `chip-build:1` and this is based on `15`? If other images are behind current chip-build, maybe they should be bumped as well?